dev/Generated/RadioMenuFlyoutItem.properties.cpp

```diff
@@ -69,6 +82,17 @@ void RadioMenuFlyoutItemProperties::OnIsCheckedPropertyChanged(
     winrt::get_self<RadioMenuFlyoutItem>(owner)->OnPropertyChanged(args);
 }
 
+
+void RadioMenuFlyoutItemProperties::SetAreCheckStatesEnabled(winrt::MenuFlyoutSubItem const& target, bool value)
```

Comment:
Can you please add some tests ?

Reply:
Added!

Reply:
can you push your changes ? I don't see the updates.

Reply:
Oh sorry. If I had a nickel for every time I committed and then forgot to push...